crates/cxx-qt-gen/src/generator/rust/cxxqttype.rs

```diff
@@ -8,57 +8,62 @@ use crate::{
     naming::TypeNames,
 };
 use quote::quote;
-use syn::Result;
+use syn::{Ident, Result};
 
 use super::fragment::RustFragmentPair;
 
 pub fn generate(
     qobject_ident: &QObjectNames,
     type_names: &TypeNames,
+    module_ident: &Ident,
 ) -> Result<GeneratedRustFragment> {
     let mut blocks = GeneratedRustFragment::default();
 
     let cpp_struct_ident = &qobject_ident.name.rust_unqualified();
     let rust_struct_ident = &qobject_ident.rust_struct.rust_unqualified();
+    let rust_fn = qobject_ident.cxx_qt_ffi_method("rust");
```

Comment:
If this returned a `Name` then couldn't the rust_qualified be used instead of doing `#module_ident::#rust_fn` further down?

Reply:
Refactored to Name, which became a bit larger of a refactor, so did that in a separate commit.